index.html

```diff
@@ -1808,77 +1894,252 @@ <h2>Authentication</h2>
-          <dd>
-The value of the <code>controller</code> property MUST be a valid <a>DID</a>
-or an array of valid <a>DIDs</a>. The corresponding <a>DID document</a>(s)
-SHOULD contain authorization relations that explicitly permit the use of
-certain verification methods for specific purposes.
-          </dd>
-      </dl>
+      <p>
+Example:
+      </p>
 
+      <pre class="example nohighlight" title="Key agreement property
+      containing three verification methods">
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  "id": "did:example:123456789abcdefghi",
+  <span class="comment">...</span>
+  "keyAgreement": [
+    <span class="comment">// this method can be used to perform key agreement
+    as did:...fghi</span>
+    "did:example:123456789abcdefghi#keys-1",
+    <span class="comment">// this method is *only* authorized for key 
+      agreement usage, it may not</span>
+    <span class="comment">// be used for any other verification relationship, 
+      so its full description is</span>
+    <span class="comment">// embedded here rather than using only a 
+      reference</span>
+    {
+      "id": "did:example:123#zC9ByQ8aJs8vrNXyDhPHHNNMSHPcaSgNpjjsBYpMMjsTdS",
+      "type": "X25519KeyAgreementKey2019",
+      "controller": "did:example:123",
+      "publicKeyBase58": "9hFgmPVfmBZwRvFEyniQDBkz9LmV7gDEqytWyGZLmDXE"
+    }
+  ],
+}
+  <span class="comment">...</span>
+      
+      </pre>
+    </section>
+      
+    <section>
+      <h3>capabilityInvocation</h3>
       <p>
-Each <a>DID method</a> MUST define how authorization and delegation are
-implemented, including any necessary cryptographic operations.
+The <code>capabilityInvocation<code> property is used to express a
+<a>verification relationship</a> which an entity can use to invoke
+capabilities as the <a>DID Subject</a> or on behalf of the <a>DID
+subject</a>. A capability is an expression of an action that the <a>DID
+subject</a> is authorized to take. The <em>verifier</em> of a
+<code>capabilityInvocation<code> attempt can check proof of an invocation of a
+capability on behalf of the <a>DID subject</a> by checking if the
+<a>verification method</a> used with the proof is contained in the
+<code>capabilityInvocation<code> property of the <a>DID Document</a>.
+      </p>
+                  
+      <p class="note" title="Uses of capabilityInvocation">
+It is up to a <a>verifier</a> to validate that the <a>verification method</a>
+used when presenting a capability is valid and is associated with the
+<code>capabilityInvocation</code> property. An example of when this property
+is useful is when a <a>DID Subject</a> chooses to invoke their capability to
+start a vehicle through the combined usage of a <a>verification method</a> and
+the "canStartCar" capability. In this example, the vehicle would be the 
+<em>verifier</em> and would need to verify the <a>verification method</a> 
+exists in the <code>capabilityInvocation</code> property. 
```

Comment:
```suggestion
exists in the <code>capabilityInvocation</code> property associated with the invoker 
of the capability. 
```

We're going to have to drag in a number of definitions from the zcap spec... trying desperately to not do that.

Reply:
Same here. Most of my understanding of these concepts came from reading that doc, so if there's easier ways to introduce examples without these concepts and terminology I'd way prefer to go in that direction.

Reply:
A lot of these concepts are generic, not specific to ZCAP-LD.

A capability is just an authorization to take some action. Someone acting as or on behalf of a DID subject (the "invoker") may invoke it by generating a cryptographic proof whereby a verifier must ensure that the verification method expressed in the proof is also expressed in a `capabilityInvocation` verification relationship in the subject's DID Document. This is as generic as generating and verifying cryptographic proofs for authentication and assertion methods -- here it is for taking actions/performing operations you're authorized to take.